pkg/csi/controller_server.go

```diff
@@ -55,8 +59,28 @@ func NewControllerServer(coreClient ctlv1.Interface, virtClient kubecli.Kubevirt
 	}
 }
 
+func (cs *ControllerServer) isStorageClassValid(storageClassName string) bool {
+	logrus.Infof("Prepare to check the associated StorageClass: %s", storageClassName)
+	storageClassInst, err := cs.storageClient.StorageClass().Get(storageClassName, metav1.GetOptions{})
+	if err != nil {
+		errReason := errors.ReasonForError(err)
+		switch errReason {
+		case metav1.StatusReasonForbidden:
+			logrus.Infof("No permission, skip checking. err: %+v", err)
+			return true
+		default:
+			logrus.Infof("Non-allowed error, fail here. err: %+v", err)
+			return false
+		}
```

Comment:
Prefer to return `error` instead when it is not an RBAC issue.

Reply:
done, thanks for making it more clear with errors